src/rules/__tests__/no-standalone-expect.test.ts

```diff
@@ -20,7 +20,6 @@ ruleTester.run('no-standalone-expect', rule, {
     'it("an it", () => expect(1).toBe(1))',
     'const func = function(){ expect(1).toBe(1); };',
     'const func = () => expect(1).toBe(1);',
-    'expect.hasAssertions()',
```

Comment:
?

Reply:
[This has become an error](https://github.com/jest-community/eslint-plugin-jest/pull/1173/files#diff-4d19ed2eeb02828caa2177ad7b043e207d6d773e56452b5dde658bfe5f7a17b6R65), since it's a standalone `expect`.